src/components/Header/styles.ts

```diff
@@ -6,67 +6,67 @@ import IconButton from '@material-ui/core/IconButton';
 import colors from '../../utils/styles/colors';
 import mq from '../../utils/styles/media';
 
-export const InnerNavBar = styled(Toolbar)`
-  && {
-    justify-content: space-between;
-    align-items: center;
-    padding: 0 15px;
-  }
-`;
+export const InnerNavBar = styled(Toolbar)({
+  '&&': {
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    padding: '0 15px',
+  },
+});
 
-export const Greetings = styled('span')`
-  && {
-    margin: 0 5px 0 0;
-  }
-`;
+export const Greetings = styled('span')({
+  '&&': {
+    margin: '0 5px 0 0',
+  },
+});
 
-export const RightSide = styled(Toolbar)`
-  && {
-    display: flex;
-    padding: 0;
-  }
-`;
+export const RightSide = styled(Toolbar)({
+  '&&': {
+    display: 'flex',
+    padding: 0,
+  },
+});
 
-export const LeftSide = styled(RightSide)`
-  && {
-    flex: 1;
-  }
-`;
+export const LeftSide = styled(RightSide)({
+  '&&': {
+    flex: 1,
+  },
+});
 
-export const MobileNavBar = styled('div')`
-  && {
-    align-items: center;
-    display: flex;
-    border-bottom: 1px solid ${colors.greyLight};
-    padding: 8px;
-    position: relative;
-  }
-`;
+export const MobileNavBar = styled('div')({
+  '&&': {
+    alignItems: 'center',
+    display: 'flex',
+    borderBottom: `1px solid ${colors.greyLight}`,
+    padding: '8px',
+    position: 'relative',
+  },
+});
 
-export const InnerMobileNavBar = styled('div')`
-  && {
-    border-radius: 4px;
-    background-color: ${colors.greyLight};
-    color: ${colors.white};
-    width: 100%;
-    padding: 0px 5px;
-    margin: 0 10px 0 0;
-  }
-`;
+export const InnerMobileNavBar = styled('div')({
+  '&&': {
+    borderRadius: '4px',
+    backgroundColor: colors.greyLight,
+    color: colors.white,
+    width: '100%',
+    padding: '0px 5px',
```

Comment:
`padding: '0 5px',` to be consistent with margin definition below and elsewhere?